cfn-resources/custom-db-role/README.md

```diff
@@ -0,0 +1,59 @@
+# MongoDB::Atlas::CustomDBRole
+
+Returns, adds, edits, and removes custom database user privilege roles.
+
+## Attributes & Parameters
+
+Please consult the [Resource Docs](docs/README.md)
+
+## Local Testing
+
+The local tests are integrated with the AWS `sam local` and `cfn invoke` tooling features:
+
```

Comment:
Missing section + link to Examples folder so users know to leverage this resource example 

https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples